<launch>
  <node ns="gvg_mapper_viz" name="viz" pkg="gvg_mapper" type="gvg_mapper_viz"  output="screen" args="/odom">
    <remap from="/indoor/odom" to="/indoor/gvg/odom_combined"/>
  </node>
</launch>
